test/docker/compose.go

```diff
@@ -225,6 +226,11 @@ func (d *Compose) WithWeaviate() *Compose {
 	return d
 }
 
+func (d *Compose) WithSecondWeaviate() *Compose {
+	d.withSecondWeaviate = true
+	return d
+}
+
```

Comment:
How is this different than `WithWeaviateCluster()`?

Reply:
With weaviate cluster starts a second node that joins with the first (going from single node to multi node).
This second weaviate starts a second node not joined to the first, effectively having 2 different single node.

This allows me to test backing up one cluster and restoring to another one (using node mapping).